internal/db/schema/migrations/postgres/10/03_target.up.sql

```diff
@@ -50,4 +50,15 @@ begin;
   create trigger immutable_columns before update on target_credential_library
     for each row execute procedure immutable_columns('target_id', 'credential_library_id', 'credential_purpose', 'create_time');
 
+create view target_library
+as
+select
+    tcl.*,
```

Comment:
I believe that the current practice for views is to explicitly state the fields, which among other things ensures that if columns are added to `tcl` that the view isn't suddenly scanning wrong to the application layer structs. Mike/Jim would know best here though.